lightly_solution.ipynb

```diff
@@ -9,110 +9,90 @@
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "DATASET_PATH = /absolute/path/to/dataset_clothing\n",
-      "Cloning into '/absolute/path/to/dataset_clothing'...\n",
+      "Cloning into 'dataset_clothing'...\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
       "remote: Enumerating objects: 3839, done.\u001b[K\n",
-      "remote: Counting objects: 100% (400/400), done.\u001b[K\n",
-      "remote: Compressing objects: 100% (400/400), done.\u001b[K\n",
-      "remote: Total 3839 (delta 9), reused 385 (delta 0), pack-reused 3439\u001b[K\n",
-      "Receiving objects: 100% (3839/3839), 100.58 MiB | 22.72 MiB/s, done.\n",
       "Resolving deltas: 100% (10/10), done.\n",
-      "Updating files: 100% (3783/3783), done.\n"
+      "\u001b[01;34mdataset_clothing/validation\u001b[00m\n",
+      "├── \u001b[01;34mdress\u001b[00m [32 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mhat\u001b[00m [14 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mlongsleeve\u001b[00m [49 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34moutwear\u001b[00m [24 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mpants\u001b[00m [49 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mshirt\u001b[00m [29 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mshoes\u001b[00m [26 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mshorts\u001b[00m [25 entries exceeds filelimit, not opening dir]\n",
+      "├── \u001b[01;34mskirt\u001b[00m [12 entries exceeds filelimit, not opening dir]\n",
+      "└── \u001b[01;34mt-shirt\u001b[00m [81 entries exceeds filelimit, not opening dir]\n",
+      "\n",
+      "10 directories, 0 files\n"
      ]
     }
    ],
    "source": [
     "\"\"\"\n",
     "Set the path to the dataset.\n",
     "Here we use the clothing-small dataset and download it. It has about 4k images.\n",
+    "If you want to use your own dataset, just set the path to it.\n",
     "\"\"\"\n",
-    "\n",
     "from pathlib import Path\n",
-    "dataset_path = Path(\"./dataset_clothing\").absolute().as_posix()\n",
     "\n",
-    "os.environ[\"DATASET_PATH\"] = dataset_path\n",
-    "!echo \"DATASET_PATH =\" $DATASET_PATH\n",
+    "dataset_path = Path(\"./dataset_clothing\")\n",
+    "!git clone https://github.com/alexeygrigorev/clothing-dataset-small.git {str(dataset_path)}\n",
     "\n",
-    "!git clone https://github.com/alexeygrigorev/clothing-dataset-small.git $DATASET_PATH"
+    "# Optional: Set the dataset path to a directory with less images, so that this example finishes faster.\n",
+    "dataset_path = dataset_path / \"validation\"\n",
+    "\n",
+    "!tree --filelimit=10 {str(dataset_path)}"
```

Comment:
This won't work on mac/windows as tree is not available by default. Looking back at the issue I am not sure if it is really needed. Or maybe we can just show in a markdown codeblock how the structure should look like?